primitives/src/v1.rs

```diff
@@ -360,18 +393,9 @@ pub struct PersistedValidationData<N = BlockNumber> {
 	/// The parent head-data.
 	pub parent_head: HeadData,
 	/// The relay-chain block number this is in the context of.
-	pub block_number: N,
+	pub relay_parent_number: N,
```

Comment:
Nitpick, why do we call it `relay_storage_root` and this here `relay_parent_number`? :D

Reply:
It seemed to me that `relay_parant_storage_number` exceeds the length that I am comfort with and on the other hand, `relay_storage_root` still conveys (as it seems to me) what it is about.

I don't mind renaming it though.

Reply:
Why not just call it `relay_number`?

Reply:
I felt that this is confusing.

Reply:
`relay_parent_storage_root` would be more clear. I think referencing the fact that every parachain block has a "relay chain parent" more explicitly is a good thing, so `relay_parent_number` is good.